src/testdir/test_taglist.vim

```diff
@@ -99,8 +99,11 @@ func Test_tagfiles()
   call assert_equal(['Xtags1', 'Xtags2'], tagfiles())
 
   help
-  let tf = tagfiles()
-  call assert_equal(1, len(tf))
+  let hplang = &helplang
+  set helplang=en  " May other 'tags-xx' (e.g 'tags-ru') exist
```

Comment:
@RestorerZ you may need to find out why here some system failed in CI if changed to 'ru'

Reply:
@Shane-XB-Qian 
It's not just about the tags-ru file. If someone, for example, add a file "uganda.de", then there will be three tag files: "tags" (for English), "tags-ru" (for Russian), "tags-de" (for German).


> @RestorerZ you may need to find out why here some system failed in CI if changed to 'ru'

As I remember, in order for "helplang" to work, it requires that Vim be built with "+multi_lang" support.
Perhaps this happens, for example, when building a "TINY" version.

Reply:
> @Shane-XB-Qian It's not just about the tags-ru file. If someone, for example, add a file "uganda.de", then there will be three tag files: "tags" (for English), "tags-ru" (for Russian), "tags-de" (for German).

interesting, i am cn, not sure why it gen 'ru' to me; and yes, now (this PR) even there 100 tag files, made it only knew 'en'.

> > @RestorerZ you may need to find out why here some system failed in CI if changed to 'ru'
> 
> As I remember, in order for "helplang" to work, it requires that Vim be built with "+multi_lang" support. Perhaps this happens, for example, when building a "TINY" version.

https://github.com/vim/vim/actions/runs/8701135087

---
again, i am not planning to look into why it failed, but just make that file not block the test.